agentinstall.ps1

```diff
@@ -64,14 +64,25 @@ wget "https://vstsagentpackage.azureedge.net/agent/$agentVersion/$zip" -OutFile
 
 
 for ($i = 1; $i -le $count; $i++) {
+    Set-Location C:\a
     $agentDir = "A$i"
-    $agentName = "$ComputerName-A$i"
+    $agentName = "$ComputerName-ws2019-A$i"
     
     Expand-Archive -Path ./$zip -DestinationPath ./$agentDir
     
     cd $agentDir
         
-    .\config.cmd --unattended --url https://dev.azure.com/$account --auth PAT --token $PAT --pool "$pool" --agent $agentName --runAsService
-    
+    .\config.cmd --unattended --url https://dev.azure.com/$account --auth PAT --token $PAT --pool "$pool Agents" --agent "$agentName" --runAsService
+}
+
+for ($i = 1; $i -le $count; $i++) {
```

Comment:
Can you just leave a stating what this loop is doing differently to the other one . Very similar apart from the pool name.

Reply:
done